Assets/_MUTUAL/Screens/LoadCaseScreen.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Assets.CaseFile;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Mutual.Screens
+{
+    class LoadCaseScreen : ILoadCaseScreen
+    {
+        #region Dependencies
+
+        private readonly Project m_Project;
+
+        #endregion
+
+        #region Private Constants
+
+        private const string ParentTag = "UIParent";
+        private const string LoadCaseBTNName = "LoadCaseBTN";
+        private const string ViewBTNName = "ViewBTN";
+
+        #endregion
+
+        #region Private Members
+
+        private Button m_LoadCaseBtn;
+        private Button m_ViewBtn;
+        private Transform m_Parent;
+        private List<GameObject> m_Meshes;
+
+        #endregion
+
+        #region Constructors
+
+        public LoadCaseScreen(Project project)
+        {
+            m_Project = project;
+            m_Parent = GameObject.FindGameObjectWithTag(ParentTag).transform;
+            m_Meshes = new List<GameObject>();
+        }
+
+        #endregion
+
+        #region Public Methods
+
+        public void ActivateScreen()
+        {
+            Debug.Log("Activated");
+            PopulateUiElements();
+            AttachListeners();
+        }
+
+        public void DeactivateScreen()
+        {
+            Camera.main.gameObject.SetActive(true);
```

Comment:
@RemyaTP  I'm guessing this is temporary

Reply:
@KICO-SOURCE Yes